src/backend/_scss/application.scss

```diff
@@ -664,21 +673,26 @@ a {
     }
   }
 
-  .track-names {
-    margin-top: 16%;
-    color: black;
-  } 
+.title-legend {
+  left: 70px;
+}
 
- @media (max-width: 990px) {
-    .title-inline {
-      width: 50%;
-      float: left;
-    }
-    .track-names {
-      margin-top :0;
-    }
+.track-names {
+  color: black;
```

Comment:
@alexagr it should not be 'black', it should be $black that is a sass variable in config.scss.